app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>here are all users</h1>
\ No newline at end of file
+<%= stylesheet_link_tag 'index', media: 'all', 'data-turbolinks-track': 'reload' %>
+<h1>Users</h1>
+<div class="user-list d-flex flex-wrap">
+  <% @all_users.each do |user| %>
+    <div class="user-card m-2">
+      <%= link_to user_id_path(user.id) do %>
+        <%= render partial: 'shared/user-card', locals: { user: user } %>
+      <% end %>
+    </div>
+  <% end %>
```

Comment:
# Optional:
- I would like to suggest you use   `<%= render @users %>` instead of using `@objects.each`, for better practive